Samples/Py/OgreParticleuniverseLayer.cpp

```diff
@@ -0,0 +1,47 @@
+#include "OgreParticleuniverseLayer.h"
+#include "Ogre.h"
+
+void OgrePartUniLayer::installPlugin(Ogre::Root * root) {
```

Comment:
in contrast to btOgre, this is not really needed. You can just wrap the relevant PU classes:
- ParticleSystemManager
- ParticleSystem

Reply:
I know. I just change directly in the repository. I should add the repository as my own submodule.
I would change them.